package.json

```diff
@@ -44,7 +44,7 @@
     "postshrinkwrap": "test -z $CI && ./scripts/packagelockHttps.sh; git update-index --assume-unchanged .env",
     "prepare": "husky install",
     "preinstall": "node scripts/check-package-manager.js",
-    "start": "NODE_ENV=production node  --max-old-space-size=3500 build/server.js",
+    "start": "NODE_ENV=production node  --max-old-space-size=3500 build/server.js & PORT=7081 node ws-nextjs-app/build/standalone/server.js",
```

Comment:
I think given the way the permissions work on the server, any manual `npm` commands that are ran would require `sudo`. The `npm run start` is triggered by the `bbc/speculate` package https://github.com/bbc/speculate#npm-start-script. I tried it previously using something like `start: yarn start:express & yarn start:nextjs`, but that didn't work due to it trying to use `yarn`. `npm` may be different, but I've kept it as close to the current Express app start as possible but just using `node .`

Reply:
Thanks for the explanation. As long as it's straightforward to us to use locally (which I think it is - we'll mostly use the `dev` rather than `start` command), and as long as it works on the test environment , then it's all good.